native/src/modules/app/constants/NavigationTypes.js

```diff
@@ -50,10 +50,10 @@ export type RoutesParamsType = {|
   intro: void,
   landing: void,
   dashboard: {| ...ShareUrlType, cityCode: string, languageCode: string |},
-  categories: {| ...ShareUrlType, cityCode: string, languageCode: string |},
-  pois: ShareUrlType,
-  events: ShareUrlType,
-  news: ShareUrlType,
+  categories: {| cityCode: string, languageCode: string |},
+  pois: { },
+  events: { },
+  news: { },
```

Comment:
```suggestion
  pois: void,
  events: void,
  news: void,
```
The share url type should also be removed from the dashboard types.